app/views/admins/index.html.erb

```diff
@@ -1,9 +1,26 @@
-<%=@user.first_name%>
+Welcome <%=@user.first_name %> | <%= link_to "Facilities", show_facilities_admin_path(0), method: :'get' %>
```

Comment:
Syntax and convention is fixed.  I hard coded the id because the helper made that a required value, several other helpers had this issue.

Reply:
Okay, so instead of doing this hardcoded band-aid fix, it would be better to fix the root of the problem, which seems to be that you're using `member` instead of `collection` inside the admin route in the `routes` file. https://guides.rubyonrails.org/routing.html#adding-collection-routes

Reply:
Redid routes with collection when an id wasn't needed.